CONTRIBUTING.md

```diff
@@ -32,22 +32,91 @@ contribute:
+
+#### 2. Install NVM
+While you can install node manually, NVM makes managing your node installation much easier. Furthermore, you can have more than one version installed at the same time. 
+
+**Note:** if you use another shell, zsh for example, replace `bash` with your shell
+`curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash`
+
+this will install `nvm` and place it in your path.
+
+Open a new terminal window and install node
+
+`nvm install 10.15.3`
+
+**Warning:** `node` version 12 is not compatible with Aragon Cli, `10.15.3` works well but the latest LTS version should work.
+
+#### 3. Install Aragon CLI
+
+`npm i -g @aragon/cli`
+
+Since version 6.0.0 ipfs is not including with the cli so install it with
+
+`aragon ipfs install`
+
+#### 4. configure cli
+Finally, if you are not using frame as you signing provider (frame is highly recommended) you need to set up a private key for use with the cli.
+
+Create a new key and get some test net Eth from the rinkeby faucet
+
+Some times the ~/.aragon file is not included in the installation which is a problem because that's where your private key goes.
+
+`cd ~/.aragon`
+
+If you get file not found run the dev chain first aragon devchain and try again
+
+`aragon devchain`
+
+Once you're in the `~/.aragon` folder you need to create a file that holds your private key.
+
+Create a new file for your rinkeby key
+
+
+`nano ~/.aragon/rinkeby_key.json`
+
+This will open a blank file, copy and paste the following replacing the text with your private key
+
+```
+{
+ "rpc": "https://rinkeby.aragon.network",
+ "keys": [
+ "put-your-priv-key-here"
+ ]
+}
+```
+
+now test out your configuration by launching a DAO on rinkeby
+
+`
+dao new --environment aragon:rinkeby
+`
+
 ### Main Aragon documents/resources
 
-- <https://wiki.aragon.one/dev/>
-- <https://github.com/aragon/aragonOS/blob/docs/docs/aragonOS.md>
-- <https://github.com/aragon/hack>
-- <https://gist.github.com/izqui/2a02c337aafa225c403de9c988cb78a0>
+- <https://wiki.aragon.org/>
+- <https://hack.aragon.org/>
+- <https://hack.aragon.org/docs/aragonos-intro.html>
+- <https://hack.aragon.org/docs/tutorial>
 - <https://github.com/aragon/aragon.js/blob/master/docs/APP.md>
 - <https://github.com/aragon/aragon.js/blob/master/docs/WRAPPER.md>
 
-The WIP gist is probably the most helpful but also the most incomplete. If you follow the instructions, you'll run into some issues with some assumptions made about the APM permissions. As far as design, the app.md and wrapper.md are the most helpful for the front end, and the hack.md gives the best overview. The aragonOS.md details the solidity code and how the core of their system works.
+The [Aragon App tutorial](https://hack.aragon.org/docs/tutorial) is probably the most helpful. As far as design, the app.md and wrapper.md are the most helpful for the front end and the hack gives the best overview. The aragonOS.md details the solidity code and how the core of their system works.

```

Comment:
We don't need to do this now, but I would like to see this commentary incorporated into the bulleted list itself.